lighthouserc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "ci": {
+      "collect": {
+        "startServerCommand": "npm start",
+        "url": ["http://localhost:4000/", "http://localhost:4000/topics/", "http://localhost:4000/resources/", "http://localhost:4000/law-and-regs/", "http://localhost:4000/cases/", "http://localhost:4000/file-a-complaint/"],
+        "psiStrategy": "desktop",
+        "settings": {
+          "maxWaitForLoad": 90000
+        }
+      },
+      "assert": {
+        "assertions": {
+          "categories:performance": ["warn", {"minScore": 0.7}],
+          "categories:accessibility": ["warn", {"minScore": 1}]
```

Comment:
we should talk about if/when to upgrade these to failures

Maybe adding a regression metric of 0.4 for performance now that we know the baseline.